### PR TITLE
Sort JSON object-keys for consistency

### PR DESCRIPTION
**Pull request by mohawk2**

### CODE REVIEW

**Comment by mohawk2** [comment]
In your last commit (39ddea188b150f10ea5c504cf42e47acc2bab8f5, which included the `Changes` message "Fixed map data (South Kensington is not linked to Knightsbridge)"), you made these changes to the map data: 

```diff
            "id" : "K014"
         },
         {
-           "link" : "H036,S012",
+           "link" : "H036,G003",
            "name" : "Knightsbridge",
            "line" : "Piccadilly:33",
            "id" : "K015"
@@ -1703,7 +1703,7 @@
         },
         {
            "id" : "S012",
-           "link" : "S006,G003,K015",
+           "link" : "S006,G003",
            "name" : "South Kensington",
            "line" : "Circle:31,District:30,Piccadilly:32"
         },
```

When I run this module's tests as they currently are on `master`, they fail. With the change in this PR, they pass again.

Your change affected:
- G003 (Gloucester Road, aka GR)
- H036 (Hyde Park Corner, aka HPC)
- K015 (Knightsbridge, aka K)
- S006 (Sloane Square, aka SS)
- S012 (South Ken, aka SK)

Before your change, the K connectivity was as follows, which was correct:

```
SK - K - HPC
```

Afterwards (one right, one wrong - you're now skipping SK which is between K and GR):

```
GR - K - HPC
```

Before your change the SK connectivity was as follows, which was correct:

```
        K
       /
GR - SK - SS
```

Afterwards it is:

```
GR - SK - SS
```

In other words, you removed the Piccadilly line connection between SK and K, despite it being shown in your screenshot.

**Comment by mohawk2** [comment]
I have therefore updated this PR to undo that map change, and update the test so it passes with the re-corrected data.

**Comment by manwar** [comment]
@mohawk2 you found a bug in Map::Tube added in v3.65. I have now reverted that changes. 
Also while doing that I fixed the missing links in the London map data. 
In the end I fixed the unit test too. 
I have uploaded all the recent changes, so you can checkout yourself.
Apology for all the trouble.

**Comment by manwar** [comment]
Thanks for your contribution as always. @mohawk2

**Comment by mohawk2** [comment]
Good heavens, that was quick! I hope you're not staying up all night!

**Comment by manwar** [comment]
I do most Sunday night as I get the weekly challenge out on Monday morning.

**Comment by mohawk2** [comment]
The motivation for me to adjust the JSON is it will be handy for me to spot diffs with my nearly-ready updating to include the Elizabeth line. That's causing what looks like an infinite loop on loading the map, so I'm still getting something wrong.

**Comment by manwar** [comment]
Good luck, please do let me know if you find any issues.
